src/run/handlers/server.ts

```diff
@@ -112,7 +112,10 @@ export default async (request: Request, context: FutureContext) => {
 
     await adjustDateHeader({ headers: response.headers, request, span, tracer, requestContext })
 
-    setCacheControlHeaders(response.headers, request, requestContext)
+    const useDurableCache = context.flags.get('serverless_functions_nextjs_durable_cache') as
+      | boolean
+      | undefined
```

Comment:
The return value here is typed as `unknown` because flags (a bit of a misnomer; they're "variables") can be booleans, strings, and numbers.

It seems a bit extreme to `assert` the type here, which would throw at runtime, so I just went with a type assertion.